<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0" rowspan="2">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>500</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QListWidget" name="queueListWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>540</width>
       <height>400</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>240</height>
         </size>
        </property>
        <property name="title">
         <string>control read composition (untested)</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="4" column="1">
          <widget class="QLineEdit" name="uncomplexStringLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maxLength">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_12">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&lt; 10 % or &gt; 40 % is this string:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>min average quality lower than:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_13">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>as above but searching all strings
of following size:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="uncomplexCharLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maxLength">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="uncomplexWordSizeSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="skipCharLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maxLength">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>54</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>they contain this character:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="filterCheckBox">
           <property name="text">
            <string>filter reads if:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="minQualSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_11">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&gt; 30 % is this char:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>210</width>
          <height>240</height>
         </size>
        </property>
        <property name="title">
         <string>control read size (untested)</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QCheckBox" name="sizingCheckBox">
           <property name="text">
            <string>resize reads</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>47</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>chop from 5' end:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QSpinBox" name="chopLeftSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>chop from 3' end:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QSpinBox" name="chopRightSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>set max size to:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QSpinBox" name="fixedSizeSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="label_5">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Sizing is done in this order:
(i) chop 5'
(ii) chop 3'
(iii)take maximal size</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>50</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>specify the reads, the sequences which shall be removed, and the output</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>reads - input (fastq/fastq.gz):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="readsInfileLineEdit"/>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="findInputPushButton">
           <property name="text">
            <string>search file</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>filter sequences (fasta):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="fastaLineEdit"/>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="findFilterPushButton">
           <property name="text">
            <string>search file</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>reads - output (fastq/fastq.gz):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="readsOutfileLineEdit"/>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="findOutputPushButton">
           <property name="text">
            <string>search file</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Important: If input is compressed, then the output will be compressed as well.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QPushButton" name="cancelPushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>cancel processing</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>189</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>readsInfileLineEdit</tabstop>
  <tabstop>findInputPushButton</tabstop>
  <tabstop>fastaLineEdit</tabstop>
  <tabstop>findFilterPushButton</tabstop>
  <tabstop>readsOutfileLineEdit</tabstop>
  <tabstop>findOutputPushButton</tabstop>
  <tabstop>sizingCheckBox</tabstop>
  <tabstop>chopLeftSpinBox</tabstop>
  <tabstop>chopRightSpinBox</tabstop>
  <tabstop>fixedSizeSpinBox</tabstop>
  <tabstop>filterCheckBox</tabstop>
  <tabstop>minQualSpinBox</tabstop>
  <tabstop>skipCharLineEdit</tabstop>
  <tabstop>uncomplexCharLineEdit</tabstop>
  <tabstop>uncomplexStringLineEdit</tabstop>
  <tabstop>uncomplexWordSizeSpinBox</tabstop>
  <tabstop>queueListWidget</tabstop>
  <tabstop>cancelPushButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>sizingCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_6</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizingCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_7</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizingCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_8</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizingCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizingCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>fixedSizeSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizingCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>chopRightSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizingCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>chopLeftSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_9</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_10</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_11</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_12</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_13</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>minQualSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>718</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>skipCharLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>686</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>uncomplexCharLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>672</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>671</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>uncomplexStringLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>655</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>717</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>uncomplexWordSizeSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>695</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
